eng/pipelines/aggregate-reports.yml

```diff
@@ -100,17 +119,17 @@ stages:
           displayName: Generate Interdependency Report Data
 
         - pwsh: |
-            Copy-Item target/staging $(Build.ArtifactStagingDirectory) -Recurse
+            Copy-Item -Path target/site/apidocs -Destination $(Build.ArtifactStagingDirectory)/staging/apidocs -Force -Recurse
+            Copy-Item -Path eng/jacoco-test-coverage/target/site/test-coverage -Destination $(Build.ArtifactStagingDirectory)/staging/test-coverage -Force -Recurse
             Copy-Item eng/code-quality-reports/src/main/resources/index.html $(Build.ArtifactStagingDirectory)
             Copy-Item eng/code-quality-reports/src/main/resources/dependency-allowlist.html $(Build.ArtifactStagingDirectory)/staging
-
             Copy-Item eng/common/InterdependencyGraph.html $(Build.ArtifactStagingDirectory)/staging
             Copy-Item data.js $(Build.ArtifactStagingDirectory)/staging
+            ls -R $(Build.ArtifactStagingDirectory)/staging
```

Comment:
If this information we want to continue outputting or was it for debugging?

Reply:
yeah, I'd like to keep it for a few runs until aggregate reports pipeline consistently passes and then we can remove.